src/helpers/getReactNativeHostComponentNames.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Text, TextInput, View } from 'react-native';
+import { HostComponentNames } from '../config';
+import render from '../render';
+
+export function getReactNativeHostComponentNames(): HostComponentNames {
```

Comment:
I think we can use simpler name since we are *React Native* TL ;-)
```suggestion
export function getHostComponentNames(): HostComponentNames {
```

Reply:
My intention here was to make it clear that this function returns the host component names of the React native version of the user and not the names from the rntl config for instance. Imo getHostComponentNames is too implicit and it could very well return the names from the config. I think the distinction is important but it seems my naming has been misunderstood so we should find another one. Maybe something like `getRealHostComponentNames` or `getAutodetectedHostComponentNames` ?

Reply:
Maybe detectHostComponentNames?

Reply:
Maybe detectHostComponentNames?